front-end/app/components/map-comp.hbs

```diff
@@ -5,9 +5,14 @@
   @onClick={{this.onMapClick}}
   as |layers|
 >
+{{#if this.view}}
   <layers.tile
     @url='https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png'
   />
```

Comment:
Can we remove this if the OSM layer is working?

Reply:
The one you're indicating is the Satellite tiles, so I think we'll need to leave it.